services/brig/test/integration/API/User/Client.hs

```diff
@@ -121,7 +121,7 @@ tests _cl _at opts p db b c g =
       test p "get/head nonce/clients" $ testNewNonce b,
       testGroup
         "post /clients/:cid/access-token"
-        [ test p "success" $ testCreateAccessToken b,
+        [ test p "invalid values" $ testCreateAccessTokenInvalidValues b,
```

Comment:
Is there no happy path test possible anymore?

Reply:
Unfortunately not. There is no simple way to create a valid DPoP proof.